_data/tools.json

```diff
@@ -162,9 +162,9 @@
       "type": "DAST"
    },
    {
-      "title": "Indusface Website Security Scan",
-      "url": "https://www.indusface.com/website-security-scan.php",
-      "owner": "AppTrana",
+      "title": "AppTrana",
+      "url": "https://www.indusface.com/website-security-scan.php?ref=owasp",
```

Comment:
Trackers on URLs are really not appreciated by the OWASP community 😁

Reply:
Noted. and removed the trackers on url please check and update it.